examples/messages.ts

```diff
@@ -11,9 +11,10 @@ const mySdk = new SDK({
 });
 
 const token = "<token>";
+const domainId = "<domainId>";
 
 mySdk.messages
-  .Send("<channelId>", "<message>", "<clientKey>")
+  .Send(domainId, "<topic>", "<secret>", "<message>")
```

Comment:
Let's keep secret the last param. Sorry about reordering comments. Rule of thumb: keep context, domain, and topic first, secrets/tokens last.